assets/_sass/layout/_footer.scss

```diff
@@ -84,6 +90,13 @@
           position: relative;
           top: -3px;
         }
+
+        &:before {
+          margin-left: 0;
+          @include breakpoint('medium') {
+            margin-left: 0.2em;
```

Comment:
Let's use rems here, not ems. So it sizes consistently with the rest of the site.